packages/nextjs/src/config/withSentryConfig.ts

```diff
@@ -379,7 +379,7 @@ function getFinalConfigObject(
             releaseName,
             routeManifest,
             nextJsVersion,
-            useRunAfterProductionCompileHook: userSentryOptions._experimental?.useRunAfterProductionCompileHook,
+            useRunAfterProductionCompileHook: userSentryOptions?.useRunAfterProductionCompileHook,
```

Comment:
l: no strong feelings but theoretically we could continue to silently support the experimental value. Given that this option wasn't available for long (I think at least), actually removing the type sounds fine to me.

```suggestion
            useRunAfterProductionCompileHook: userSentryOptions?.useRunAfterProductionCompileHook ?? userSentryOptions._experimental?.useRunAfterProductionCompileHook,
```

wdyt? feel free to overrule me :)

(the suggestion probably needs further TS adjustments)